src/test/java/io/pinecone/PineconeTest.java

```diff
@@ -0,0 +1,239 @@
+package io.pinecone;
```

Comment:
This is essentially a refactor of what was in `PineconeIndexOperationClientTest` which was completely commented out. I think these could be expanded and improved further down the line, for now I just restored the basic validation that was in place previously.

It's slightly different now since we rely on generated classes.